src/content/docs/release-notes/synthetics-release-notes/job-manager-release-notes/job-manager-release-206.mdx

```diff
@@ -5,4 +5,4 @@ version: '206'
 ---
 
 ### Fixes
-* Skip querying for the instanceID if the Metadata field is disabled to avoid Server Side Request Forgery attacks that can leak information
+* Skipped querying for the instanceId if the Metadata field is disabled to avoid Server Side Request Forgery attacks that can leak information
```

Comment:
```suggestion
* Skipped querying for the` instanceId` if the Metadata field is disabled to avoid Server Side Request Forgery attacks that can leak information
```